data/bactrian.txt

```diff
@@ -168,15 +169,15 @@ mission "Bactrian Unlock [1]"
 			
 			label next
 			`	After hearing this news, many of the faces in the group turn grim as they realize that they will need to fight to retrieve the Star Queen. `
-			`	"We will need to get a rescue fleet ready. Soldiers, ready your ships!" The group responds with a collective "Sir, yes sir!" before running off to their ships.`
+			`	"We will need to get a rescue fleet ready. Guards, ready your ships!" The group responds with a collective "Sir, yes sir!" before running off to their ships.`
 			`	Before leaving for his own ship, the man walks up to you and hands you 100,000 credits.`
 			`	"That's for finding the Star Queen. Now, Captain <last>, would you be willing to help us? If the Star Queen is as heavily defended as you say it is then we're going to need every ship we can get."`
 			choice
 				`	"I'd be glad to."`
 				`	"I would rather let the professionals handle things. Good luck."`
 					decline
 			
-			`	"Thank you so much. The plan is to rendezvous with a number of warships on Memory and then head to Alnitak. My superiors received word from the commander on Farpoint that there are a number of Navy ships at the ready to avenge the ships they lost, so I expect we'll pick up some help there when we refuel. After the fleet is mustered, we go to Arneb." He smiles briefly, adding, "Just make sure that the <npc> makes it onto the planet. She'll be carrying the landing crew, along with myself."`
+			`	"Thank you so much. The plan is to rendezvous with a number of warships on Memory and then head to Alnitak. My superiors received word from the admiral on Farpoint that there are a number of Navy ships at the ready to avenge the ships they lost, so I expect we'll pick up some help there when we refuel. After the fleet is mustered, we go to Arneb." He smiles briefly, adding, "Just make sure that the <npc> makes it onto the planet. She'll be carrying the landing crew, along with myself."`
 			`	The communications device on his belt squawks, and he quickly responds, giving orders to the fleet that will be joining you on the mission. Before you can return to your ship, he says, "Also, since we'll be going into battle together, I might as well introduce myself. I'm Deep Lieutenant Shane Paris. Good luck fighting; we'll meet you in orbit."`
```

Comment:
> I'm Deep Lieutenant Shane Paris.

Seems… unrealistic? Does anyone introduce themselves as "I'm Russian junior sergeant", or "I'm American brigade general?" It would be rather "I am lieutenant Shane Paris of the Deep Security forces."